db/connection.js

```diff
@@ -0,0 +1,55 @@
+import mariadb from "mariadb";
+let host = "localhost"
+let user =  "root"
+let password  = "secure1234"
+let database = "quickOrder"
```

Comment:
because this is a public repository now you might want to move DB credentials to another file that will be ignored by git and provide an example file for credentials

Reply:
okay, I modified it. You would like to notify you that you need to create ./.env file by yourself.

Reply:
yea sure